diff --git a/node_modules/leaflet/dist/leaflet-src.js b/node_modules/leaflet/dist/leaflet-src.js
index eeb30df..714a21f 100644
--- a/node_modules/leaflet/dist/leaflet-src.js
+++ b/node_modules/leaflet/dist/leaflet-src.js
@@ -2541,6 +2541,7 @@
   // using CSS translate or top/left positioning depending on the browser
   // (used by Leaflet internally to position its layers).
   function setPosition(el, point) {
+  	if (!el) { return; }
 
   	/*eslint-disable */
   	el._leaflet_pos = point;
@@ -2560,6 +2561,7 @@
   	// this method is only used for elements previously positioned using setPosition,
   	// so it's safe to cache the position for performance
 
+  	if (!el) { return new Point(0, 0); }
   	return el._leaflet_pos || new Point(0, 0);
   }
 
